test/setup/index.ts

```diff
@@ -37,6 +38,15 @@ export async function removeMonitor(): Promise<void> {
   }
 }
 
+export async function removeLocalContainerRegistry(): Promise<void> {
+  try {
+    await exec('docker rm kind-registry --force');
+    await exec('docker network prune --force');
```

Comment:
I think we should remove the `kind` network specifically and not _all_ unused networks on the system, just to be a bit more accurate and not touch other things that we didn't create

Reply:
Totally makes sense!